test/libsolidity/syntaxTests/pragma/missing_disjunction_token.sol

```diff
@@ -0,0 +1,3 @@
+pragma solidity 0.4.1 - 0.6.2 0.8.17;
```

Comment:
Can you add also these?
- `pragma solidity 0.8.17 0.4.1 - 0.6.2;` (invalid)
- `pragma solidity 0.8.17 || 0.4.1 - 0.6.2;` (valid)
- `pragma solidity 0.4.0 - 0.4.1 0.4.1 - 0.6.2;` (invalid)
- `pragma solidity 0.4.0 - 0.4.1 || 0.4.1 - 0.6.2;` (valid)